scalatest/src/main/scala/org/scalatest/Entry.scala

```diff
@@ -39,7 +39,7 @@ package org.scalatest
  * @param value the value of this entry
  */
 // SKIP-SCALATESTJS,NATIVE-START
-case class Entry[K, V](key: K, value: V) extends java.util.Map.Entry[K, V] {
+case class Entry[K, V](key: K | Null, value: V | Null) extends java.util.Map.Entry[K, V] {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.